<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySimIIGui::SolverSettings</class>
 <widget class="QWidget" name="MySimIIGui::SolverSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>695</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1666667</width>
    <height>1666667</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Solver settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Solver options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>LinearDirectMethod</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>ExecutionWhenCoupling</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="Label_Solvers">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Equation:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="CB_LinearDirectMethod"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="DirectiveSolver">
        <property name="maximumSize">
         <size>
          <width>115</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>DirectiveSolver</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="IterativeSolver">
        <property name="maximumSize">
         <size>
          <width>115</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>IterativeSolver</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>PreconditioningMethod</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="CB_ExecutionWhenCoupling"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>LinearIterativeMethod</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="CB_BiCGStablDegree"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="CB_LinearIterativeMethod"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="CB_PreconditioningMethod"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="ComboBox_Equations">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>EM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Heat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EM+Heat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CF+Heat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EM+CF+Heat</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>BiCGStablDegree</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Iteration parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QLineEdit" name="LE_LinearTolerance"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>NewtonTolerance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>LinearMaxIter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LE_LinearMaxIter"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>NonLinearTolerance</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="LE_NonLinearTolerance"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>LinearTolerance</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>NonLinearMaxIter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>SteadyTolerance</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>NonLinearRelax</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="LE_NewtonTolerance"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="LE_NewtonStartIter"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>NewtonStartIter</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="LE_NonLinearMaxIter"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LE_SteadyTolerance"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="LE_NonLinearRelax"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
